app/controllers/geocoder_controller.rb

```diff
@@ -15,8 +15,32 @@ def search
     if @params[:lat] && @params[:lon]
       @sources.push "latlon"
       @sources.push "osm_nominatim_reverse"
+
+      # get query parameters
+      lat = params[:lat]
+      lon = params[:lon]
+      zoom = params[:zoom]
+
+      # create URL for nominatim reverse search
+      @cached_osm_nominatim_parameters = "reverse?format=html&lat=#{lat}&lon=#{lon}&zoom=#{zoom}&accept-language=#{http_accept_language.user_preferred_languages.join(',')}"
```

Comment:
This, and the equivalent code for forward searches, is just duplicating the work that the search methods do.

It would be better to move that code into private `nominatim_reverse_url` and `nominatim_url` methods that can then be used in both places.